parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -121,33 +112,30 @@ installGPUDrivers() {
 }
 
 installSGXDrivers() {
-    local VERSION
-    VERSION=$(grep DISTRIB_RELEASE /etc/*-release| cut -f 2 -d "=")
-    case $VERSION in
+    local URL=""
+    case $UBUNTU_RELEASE in
     "18.04")
-        SGX_DRIVER_URL="https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntuServer18.04/sgx_linux_x64_driver_1.21.bin"
+        URL="https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntuServer18.04/sgx_linux_x64_driver_1.21.bin"
         ;;
     "16.04")
-        SGX_DRIVER_URL="https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntuServer16.04/sgx_linux_x64_driver_1.21.bin"
+        URL="https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntuServer16.04/sgx_linux_x64_driver_1.21.bin"
         ;;
     "*")
-        echo "$VERSION is not supported"
-        exit 1
+        exit $ERR_SGX_DRIVERS_INSTALL_TIMEOUT
         ;;
     esac
 
     local PACKAGES="make gcc dkms"
     wait_for_apt_locks
     retrycmd_if_failure 30 5 3600 apt-get -y install $PACKAGES  || exit $ERR_SGX_DRIVERS_INSTALL_TIMEOUT
 
-    local SGX_DRIVER
-    SGX_DRIVER=$(basename $SGX_DRIVER_URL)
+    local DRIVER=$(basename $URL)
```

Comment:
These could be one statement BTW:
`local DRIVER=$(basename $URL) OE_DIR=/opt/azure/containers/oe`

Doesn't save that many characters, but TIL more about `local`. 📖